package.json

```
@@ -24,14 +24,14 @@
     "lint:fix": "eslint ./test --fix",
     "migrate": "npx truffle migrate",
     "migrate-flattened": "npm run flatten && npx truffle migrate --config flattened-config.js",
-    "postinstall": "mkdir -p build/contracts >/dev/null 2>1; cp node_modules/witnet-ethereum-bridge/build/contracts/MockWitnetBridgeInterface.json build/contracts",
+    "postinstall": "mkdir -p build/contracts >/dev/null 2>1; cp node_modules/witnet-ethereum-bridge/build/contracts/MockWitnetRequestsBoard.json build/contracts",
```

I just realized there was a bug in this line, a missing `&`. Otherwise it will create an empty `1` file. :smile: 

```suggestion
    "postinstall": "mkdir -p build/contracts >/dev/null 2>&1; cp node_modules/witnet-ethereum-bridge/build/contracts/MockWitnetRequestsBoard.json build/contracts",
```